src/app/examples/card-example/card-example.component.tns.html

```diff
@@ -1,15 +1,27 @@
-<StackLayout>
+<StackLayout orientation="vertical" width="100%" height="100%">
   <kirby-card>
-    <kirby-card-header [title]="'Titel'" [subtitle]="'Subtitel'">
-    </kirby-card-header>
+    <kirby-card-header [title]="'Titel'" [subtitle]="'Subtitel'"></kirby-card-header>
 
     <!-- Card content example: -->
-    <ScrollView>
-      <StackLayout class="card-content">
-        <Label text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Qui quas quibusdam eligendi, debitis nemo fuga quia accusamus doloremque sit reprehenderit!" textWrap="true"></Label>        
-      </StackLayout>
-    </ScrollView>
+     <ScrollView>
```

Comment:
You touched it so now you own it 😄  

But do we really need a `StackLayout` in `ScrollView` here? A `GridLayout` should be enough.

It is just a minor thing, so you can ignore it if you don't have the time to fix it.